### ISSUE TITLE
Problem with Latex babel

### ISSUE THREAD

**Issue by juhucoding** [JensErat/pandoc-scrlttr2#3]
Does not work. It seems to me, there is a problem in the babel-package:

This was the error I got for the fresh install. For the languages I also tried de-DE and ngerman,

Thank you for your effort.

`\>\> git clone https://github.com/JensErat/pandoc-scrlttr2.git
Cloning into 'pandoc-scrlttr2'...
remote: Counting objects: 40, done.
remote: Total 40 (delta 0), reused 0 (delta 0), pack-reused 40
Unpacking objects: 100% (40/40), done.
Checking connectivity... done.

\>\> ./panletter example-letter.md
! Package babel Error: You haven't specified a language option.

See the babel package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.333 ...ry to proceed from here, type x to quit.}

pandoc: Error producing PDF from TeX source`

**Comment by JensErat** [comment]
Which pandoc version do you use (`pandoc --version | head -1`)? There was a rather recent change in language attribute parsing (with pandoc 1.14 or 1.15), and this repository already reflects this change.

You can also enforce LaTeX output as intermediate result (`panletter example-letter.md -o example-letter.tex`) to verify what language was passed to LaTeX.

I've been using the template together with `lang: de-DE` successfully, also the `example-letter.md` builds fine.

**Comment by juhucoding** [comment]
Wow, that was a fast response. 

I'm using brew on OS X, Pandoc Version 1.13.2. No update available via brew atm. 

When generating the tex-file via pandoc, I had to manually insert ngerman in `\usepackage[shorthands=off,main=ngerman]{babel}` like so. 

I'm going to sleep now, but check back tomorrow ...

**Comment by JensErat** [comment]
Look at the Formula, [pandoc 1.16 should be available in homebrew](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/pandoc.rb). I'm currently installing it myself (was on 1.15.1 up to now). Behavior definitely changed since 1.13 (pandoc is still evolving rather fast), I'd definitely recommend an upgrade.

I'll report back tomorrow whether 1.16 is working fine (I didn't observe any relevant changes in the changelog) and fix any issues with the newest release.

Since pandoc 1.13, some parts of the template have been modified. If you're really stuck with the old pandoc release, you might want to try an older patch level of the scrlttr2-template, that might still work out.

**Comment by JensErat** [comment]
I just verified against pandoc 1.16.0.2 from Debian unstable (Homebrew/Linuxbrew didn't compile for me), which worked fine, as it does with 1.15.

If you're really stuck with that old pandoc release, try falling back to commit f2f44b2faebe24941d7835d384f856228fe4fe8e (which is the last one before I refactored the template file for Pandoc 1.15).

    git checkout f2f44b2faebe24941d7835d384f856228fe4fe8e

I'm closing the issue for now; maintaining separate sets of templates for different pandoc versions is out of scope for me.
